### PR TITLE
Various constants, functions and COM interfaces added

### PR DESCRIPTION
**Pull request by rozmansi**

### CODE REVIEW

File: user32.go [hunk 1]
```diff
@@ -1854,6 +1854,7 @@ var (
 	releaseCapture              *windows.LazyProc
 	releaseDC                   *windows.LazyProc
 	removeMenu                  *windows.LazyProc
+	deleteMenu                  *windows.LazyProc
```
Comment (lxn) [inline user32.go:1857]: Please keep functions sorted.

Comment (lxn) [reply user32.go:1857]: Wow, that's quite some stuff. Thanks.

**Comment by lxn** [comment]
Wow, that's quite some stuff. Thanks.

File: user32.go (hunk 1 above)
Comment (rozmansi) [reply user32.go]: Sorted them and force-pushed. Thanks for your feedback.

**Pull request merged**